fix templating compilation error, let type inference do its job
authors: Kenneth Hoste (HPC-UGent), Ward Poelmans (VUB)
--- tophat-2.1.1/src/tophat_reports.cpp.orig	2017-05-16 11:24:21.776900497 +0200
+++ tophat-2.1.1/src/tophat_reports.cpp	2017-05-16 11:25:31.958523978 +0200
@@ -2705,7 +2705,7 @@
 				junction_stat.gtf_match = true;
 				junction_stat.accepted = true;
 
-				gtf_junctions.insert(make_pair<Junction, JunctionStats>(Junction(ref_id, left_coord, right_coord, antisense), junction_stat));
+				gtf_junctions.insert(make_pair(Junction(ref_id, left_coord, right_coord, antisense), junction_stat));
 			}
 		}
 		fprintf(stderr, "Loaded %d GFF junctions from %s.\n", (int)(gtf_junctions.size()), gtf_juncs.c_str());
